pkg/kubernetes/persistentvolumeclaim/v1alpha1/build.go

```diff
@@ -172,6 +172,13 @@ func (b *Builder) WithCapacity(capacity string) *Builder {
 	return b
 }
 
+// WithVolumeMode sets the volumeMode field in PVC with provided arguments
```

Comment:
this can be made private and called from within in this PV based on path or block device.